x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_form/form_row_setting.tsx

```diff
@@ -15,8 +15,14 @@ import {
 } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import React from 'react';
+import styled from 'styled-components';
 import { FormRowOnChange } from './';
 import { SettingsRow } from '../typings';
+import { CodeEditor } from '../../../../../../../../src/plugins/kibana_react/public';
+
+const FixedHeightDiv = styled.div`
```

Comment:
out of curiosity, why do you need a fixed height div when you have defined the code editor `height="300px"`

Reply:
This is happening when the container div is removed, so followed what was already defined in [fleet](https://github.com/elastic/kibana/blob/fd82feae08359e7bc529bc28ce00e012dff3805d/x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/package_policy_input_var_field.tsx#L63) (same with the params for the code editor)
![image](https://user-images.githubusercontent.com/5831975/151963138-d30b9316-c20f-45c5-8a5b-48ef18533469.png)

Reply:
good to know, thanks